src/components/home/Home.js

```diff
@@ -16,11 +19,16 @@ function Home() {
 
     useEffect(() => {
         getToDos();
-    }, []);
+    }, [user]);
+
+    useEffect(() => {
+        bottomRef.current?.scrollIntoView({ behavior: "smooth" });
+    }, [isFormOpen]);
 
     async function getToDos() {
+        if (!user.id) return;
```

Comment:
in the future, we might want to have a fallback here ('there are currently no to do's, would you like to create one?' or something similar)